service/src/main/java/uk/gov/hmcts/reform/fpl/service/FurtherEvidenceNotificationService.java

```diff
@@ -21,14 +21,35 @@
 @Service
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class FurtherEvidenceNotificationService {
-    private final InboxLookupService inboxLookupService;
+    private final LocalAuthorityRecipientsService localAuthorityRecipients;
     private final NotificationService notificationService;
 
     private final FurtherEvidenceUploadedEmailContentProvider furtherEvidenceUploadedEmailContentProvider;
 
-    public Set<String> getLocalAuthoritySolicitorEmails(CaseData caseData) {
-        return inboxLookupService.getRecipients(
-            LocalAuthorityInboxRecipientsRequest.builder().caseData(caseData).build());
+    public Set<String> getLocalAuthoritiesRecipients(CaseData caseData) {
+        final RecipientsRequest recipientsRequest = RecipientsRequest.builder()
+            .caseData(caseData)
+            .build();
+
+        return localAuthorityRecipients.getRecipients(recipientsRequest);
```

Comment:
is it worth extracting "localAuthorityRecipients.getRecipients(recipientsRequest)" into a method?